src/Storage/Controllers/InstancesController.cs

```diff
@@ -276,6 +276,35 @@ public async Task<ActionResult<Instance>> Get(int instanceOwnerPartyId, Guid ins
             }
         }
 
+        /// <summary>
+        /// Gets process info relevant to authorization
+        /// </summary>
+        /// <param name="instanceOwnerPartyId">The party id of the instance owner.</param>
+        /// <param name="instanceGuid">The id of the instance to retrieve.</param>
+        /// <returns>Authorization info.</returns>
+        [HttpGet("AuthInfo/{instanceOwnerPartyId:int}/{instanceGuid:guid}")]
```

Comment:
URLs are case insensitive in most cases, but I prefer that we define routes in lowercase.

Reply:
Done